### PR TITLE
WIP: reago

### PR DESCRIPTION
**Pull request by HassanAmr**
* [x] I have read the guidelines above.
* [x] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by HassanAmr** [comment]
Hi @bgruening,
Sorry for creating a WIP pull request, but I don't know why the patch is not working since it works locally (without conda), and I wanted to ask your help in the matter.

**Comment by HassanAmr** [comment]
Is there a better way to discuss workflows without doing pull requests?

**Comment by bgruening** [comment]
There seems to be a problem with the setup.py file?

/opt/conda/conda-bld/_b_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeho/bin/python: can't open file 'setup.py': [Errno 2] No such file or directory

**Comment by HassanAmr** [comment]
Yeah, I checked the working directory of conda build, the patch doesn't generate it.
but when I run `patch < reago.patch`, the file generates just fine.

**Comment by bgruening** [comment]
You could simply put this setup.py in this repository and copy the file from RECIPE_DIR to SRC_DIR isn't it?

**Comment by HassanAmr** [comment]
oh, ok. That will work. Should I then delete it?

**Comment by bgruening** [comment]
what do you want to delete? Just push to your branch and it will update this PR. The patch can be removed, yes.

**Comment by HassanAmr** [comment]
No, I mean should I delete setup.py after running it?

**Comment by bgruening** [comment]
No, it will not end up in $PREFIX. So no need for removing it.

**Comment by HassanAmr** [comment]
Ok, thanks.
What should I put for the License?
Also, how can I discuss the other WIP workflows without a PR, or should I just do the pull requests for the them as well?

File: recipes/reago/meta.yaml [hunk 1]
```diff
@@ -13,9 +13,19 @@ requirements:
   build:
     - python
     - setuptools
+    - os
```
Comment (bgruening) [inline recipes/reago/meta.yaml]: No. This is not working. Please test this locally. os, sys, time, operator are all in the standard lib of pyhon. Networkx is a module that needs to be installed.

Comment (HassanAmr) [inline recipes/reago/meta.yaml]: Sorry for uploading it faulty (I was switching machines), it does indeed fail locally too.
So, I only need to include Networkx as a dependency, and leave the rest out?

**Comment by bgruening** [comment]
@HassanAmr looks good! Please try to upstream this patch :)

**Comment by HassanAmr** [comment]
thanks :) Will do

**Pull request merged**